docs/solid-testing-library/api.mdx

```diff
@@ -0,0 +1,180 @@
+---
+id: api
+title: API
+sidebar_label: API
+---
+
+Due to being inspired by the [preact-testing-library][preact-docs] you can check
+its page for more information.
+
+[preact-docs]: ../preact-testing-library/intro.mdx
+
+There are several key differences, to be aware of.
+
+## Differences from the Preact Testing Library
```

Comment:
We usually also keep a table of contents for all the API's in this file with links (these can be auto generated AFAIR) :)

Reply:
adding a Toc

Reply:
Thank you :)